lib/components/artwork_grids/generic_grid.js

```diff
@@ -117,7 +130,6 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
   },
   section: {
-    flex: 1,
```

Comment:
This was making the sections expand to fill the screen, making for a really weird transition. Eventually the layout would settle into 3 or 4 columns but with this removed, it looks much better.